app/helpers/zizia/application_helper.rb

```diff
@@ -30,5 +30,11 @@ def collections_for_select
     def collections?
       !ActiveFedora::SolrService.query('has_model_ssim:Collection').empty?
     end
+
+    def status_icon(status)
+      # rubocop:disable Rails/OutputSafety
+      return "<span class='glyphicon glyphicon-ok-sign text-success'></span>".html_safe if status == 'attached'
+      "<span class='glyphicon glyphicon-question-sign'></span>".html_safe
```

Comment:
When we use icons semantically in this way, we should generally include an `aria-label` see https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility

Reply:
@mark-dce There's a rubocop/bixby rule we have for this: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/GuardClause

https://github.com/samvera-labs/bixby/blob/master/bixby_default.yml#L129

It will fail if you use an if / else block when you could have used a guard clause.